dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -22,6 +22,15 @@
 			status = "disabled";
 			label = "UART_9";
 		};
+
+		usart10: serial@40011C00 {
```

Comment:
I think `C` should be `lowercase`.
Recent commits are shown in lowercase.
https://github.com/zephyrproject-rtos/zephyr/commit/aace0063ecf6e3069494834d4a9f98422dffb12d#diff-c401060329ae8e5b90389bec57dfa4f088d9db73a260b07bf35ea87ef8362650
https://github.com/zephyrproject-rtos/zephyr/commit/8d09a4942834d5a23a81401748a8b9d65b171208#diff-c401060329ae8e5b90389bec57dfa4f088d9db73a260b07bf35ea87ef8362650

Reply:
Sharp eyes to spot that. Is it part of the device tree or zephyr convention?

Reply:
There are [Coding Guidelines](https://docs.zephyrproject.org/latest/contribute/index.html#coding-guidelines).
There is no direct content related to the device tree, but there are a number of commits worth referencing.